content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,225 @@
+        rules:
+          - operations: [ "CREATE" ]
+            apiGroups: [""]
+            apiVersions: ["v1"]
+            resources: ["pods"]
+        failurePolicy: Fail
+        namespaceSelector:
+          matchLabels:
+            istio-injection: enabled
+    {{< /text >}}
+
+<!-- TODO (lei-tang): improve the UX for obtain ValidatingWebhookConfiguration -->
+1.  From `istio-webhook-config.yaml`, search `'kind: ValidatingWebhookConfiguration'` and save
+the `ValidatingWebhookConfiguration` of Galley to `galley-webhook.yaml`. The following
+is a `ValidatingWebhookConfiguration` in an example `istio-webhook-config.yaml`.
+
+    {{< text yaml >}}
+    apiVersion: admissionregistration.k8s.io/v1beta1
+    kind: ValidatingWebhookConfiguration
+    metadata:
+      name: istio-galley
+      labels:
+        app: galley
+        chart: galley
+        heritage: Tiller
+        release: istio
+        istio: galley
+    webhooks:
+      - name: pilot.validation.istio.io
+        clientConfig:
+          service:
+            name: istio-galley
+            namespace: istio-system
+            path: "/admitpilot"
+          caBundle: ""
+        rules:
+          - operations:
+            - CREATE
+            - UPDATE
+            apiGroups:
+            - config.istio.io
+            ... SKIPPED
+        failurePolicy: Fail
+        sideEffects: None
+    {{< /text >}}
+
+1.  Enable webhook configurations through `istioctl`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook enable --validation --webhook-secret dns.istio-galley-service-account \
+        --namespace istio-system --validation-path galley-webhook.yaml \
+        --injection-path sidecar-injector-webhook.yaml
+    {{< /text >}}
+
+1.  Check the Sidecar Injector webhook is working:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-injection; kubectl label namespaces test-injection istio-injection=enabled
+    $ kubectl run --generator=run-pod/v1 --image=nginx nginx-app --port=80 -n test-injection
+    $ kubectl get pod -n test-injection
+    {{< /text >}}
+
+    The output from the `get pod` command should show the following output (`2/2` means that
+    the Sidecar Injector webhook injects a sidecar to the example pod):
+
+    {{< text plain >}}
+    NAME    READY   STATUS    RESTARTS   AGE
+    nginx-app   2/2     Running   0          10s
+    {{< /text >}}
+
+1.  Check the validation webhook is working:
+
+    {{< text bash >}}
+    $ cat <<EOF > ./invalid-gateway.yaml
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: invalid-gateway
+    spec:
+      selector:
+        # DO NOT CHANGE THESE LABELS
+        # The ingressgateway is defined in install/kubernetes/helm/istio/values.yaml
+        # with these labels
+        istio: ingressgateway
+    EOF
+    $ kubectl create namespace test-validation
+    $ kubectl apply -f invalid-gateway.yaml -n test-validation
+    {{< /text >}}
```

Comment:
Delete the `test-validation` namespace!

Reply:
The deleting of test namespaces is part of the cleanup in the guide: "you may delete the testing cluster created at the beginning of this tutorial."